sw/host/tests/chip/rv_core_ibex_isa/src/main.rs

```diff
@@ -46,10 +46,10 @@ fn ibex_isa_smoke_test(opts: &Opts, transport: &TransportWrapper) -> Result<()>
     uart.clear_rx_buffer()?;
 
     // Load SRAM program
-    match opts
-        .sram_program
-        .load_and_execute(&mut *jtag, ExecutionMode::JumpAndWait(Duration::from_secs(5)))?
-    {
+    match opts.sram_program.load_and_execute(
```

Comment:
Unrelated change?

Reply:
Yeah this is unrelated and went in together with the `repos.bzl` change above - this is removed now.